pkg/cli/admin/release/info.go

```diff
@@ -328,7 +335,14 @@ func replaceClusterSemanticArgs(f kcmdutil.Factory, args []string, semanticArgs
 	return args, nil
 }
 
-func findArgumentsFromCluster(f kcmdutil.Factory, args []string) ([]string, error) {
+func findArgumentsFromCluster(f kcmdutil.Factory, args []string,arch string) ([]string, error) {
+
+	//validate arch input and set default to amd64
+	//should also validate that arch is amd64, ppc64le, s390x, or arm64..but..
+	//this probably belongs somewhere else.... maybe release.go?
+	if arch == ""{
+		arch="amd64"
```

Comment:
i don't think arch needs to be specified? if it is not present, it is by default amd64.